src/test/resources/properties/special_characters.properties

```diff
@@ -0,0 +1,18 @@
+# You are reading the ".properties" entry.
+! The exclamation mark can also mark text as comments.
+# The key characters =, and : should be written with
+# a preceding backslash to ensure that they are properly loaded.
+# However, there is no need to precede the value characters =, and : by a backslash.
+website = https://en.wikipedia.org/
+language = English
+# The backslash below tells the application to continue reading
+# the value onto the next line.
+message = Welcome to \
+          Wikipedia!
```

Comment:
nice

maybe some <br> here but this will be rendered by the browser :)

Reply:
maybe change website and message text?
```
website = https://github.com/damianszczepanik/cucumber-reporting
language = English
# The backslash below tells the application to continue reading
# the value onto the next line.
message = Welcome to \
          cucumber-reporting!
```
👍